projects/js-packages/components/components/jetpack-footer/index.jsx

```diff
@@ -22,7 +22,7 @@ import JetpackLogo from '../jetpack-logo';
  * @returns {React.Component} JetpackFooter component.
  */
 export default function JetpackFooter( {
-	a8cLogoHref,
+	a8cLogoHref = 'https://www.jetpack.com',
```

Comment:
I think we can remove the `www` since it's stripped later on.

```suggestion
	a8cLogoHref = 'https://jetpack.com',
```